ctpolicy/ctpolicy_test.go

```diff
@@ -102,3 +103,113 @@ func TestLifetimeInMonths(t *testing.T) {
 		})
 	}
 }
+
+func TestGroupByLogs(t *testing.T) {
+	tests := []struct {
+		name      string
+		logGroups map[string]*LogGroupInfo
+		want      map[string]map[string]bool
+	}{
+		{
+			name: "BaseGroup",
+			logGroups: map[string]*LogGroupInfo{
+				BaseName: {
+					name: BaseName,
+					LogURLs: map[string]bool{
+						"ct.googleapis.com/aviator/":   true,
```

Comment:
Aside: it makes me wince every time I see these kind of values (non-URLs stored in a field named LogURL...)

Reply:
Do you prefer URI?

Reply:
I prefer URLs (or URIs) to actually be URLs (i.e. to have the protocol identifier)

Reply:
I can change it for the test (if that's what you want), but not at the real log-lists. Passed your comment to Rob.